underfs/s3/src/test/java/alluxio/underfs/s3/S3UtilsTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.s3;
+
+import org.jets3t.service.acl.AccessControlList;
+import org.jets3t.service.acl.CanonicalGrantee;
+import org.jets3t.service.acl.GroupGrantee;
+import org.jets3t.service.acl.Permission;
+import org.jets3t.service.model.StorageOwner;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for {@link S3Utils} methods.
+ */
+public final class S3UtilsTest {
+
+  private static final String NAME = "foo";
+  private static final String ID = "123456789012";
+  private static final String OTHER = "987654321098";
+
+  private CanonicalGrantee mOwnerGrantee;
+  private AccessControlList mAcl;
+
+  @Before
+  public void before() throws Exception {
+    // Set up owner id, name and mOwnerGrantee.
+    StorageOwner owner = new StorageOwner(ID /* id */, NAME /* display name */);
+    mOwnerGrantee = new CanonicalGrantee(ID);
+    mOwnerGrantee.setDisplayName(NAME);
+
+    // Create a mAcl with the owner.
+    mAcl = new AccessControlList();
+    mAcl.setOwner(owner);
+  }
+
+  /**
+   * Tests for {@link S3Utils#translateBucketAcl(AccessControlList, String)}.
+   */
+  @Test
+  public void translateOwnerAclTest() {
+    // Grant only READ, READ_ACP permission to the owner. Check the translated mode is 0500.
+    mAcl.grantPermission(mOwnerGrantee, Permission.PERMISSION_READ);
+    mAcl.grantPermission(mOwnerGrantee, Permission.PERMISSION_READ_ACP);
+    Assert.assertEquals((short) 0500, S3Utils.translateBucketAcl(mAcl, ID));
+    Assert.assertEquals((short) 0000, S3Utils.translateBucketAcl(mAcl, OTHER));
+
+    // Grant WRITE permission to the owner. Check the translated mode is 0700.
+    mAcl.grantPermission(mOwnerGrantee, Permission.PERMISSION_WRITE);
+    Assert.assertEquals((short) 0700, S3Utils.translateBucketAcl(mAcl, ID));
+    // Add WRITE_ACP permission to the owner. Check the translated mode is still 0700.
```

Comment:
Is each section independent? It seems like this could be broken down to a bunch of 2-line tests.